src/main/java/appeng/helpers/PatternHelper.java

```diff
@@ -119,7 +125,13 @@ public PatternHelper( final ItemStack is, final World w )
 
 			for( int x = 0; x < outTag.tagCount(); x++ )
 			{
-				final ItemStack gs = new ItemStack( outTag.getCompoundTagAt( x ) );
+				NBTTagCompound resultItemTag = outTag.getCompoundTagAt( x );
+				final ItemStack gs = new ItemStack( resultItemTag );
+
+				if( !resultItemTag.hasNoTags() && gs.isEmpty() )
+				{
+					//throw new IllegalArgumentException( "No pattern here!" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)